MoreJS/ajax.md

```diff
@@ -0,0 +1,13 @@
+ - We can (and should) cover and use `fetch()`.. and now would be a good time to discuss polyfills.
```

Comment:
I just wrote the ajax lesson yesterday, (renamed it to APIs) . I mention axios, but want to stick with native stuff for now.